docs/guides/burst_insar_product_guide.md

```diff
@@ -0,0 +1,134 @@
+{% extends "guides/insar_product_guide_template.md" %}
+
+{% block header %}
+# Sentinel-1 Burst InSAR Product Guide
+
+This document is a guide for users of Sentinel-1 Burst Interferometric Synthetic Aperture Radar (InSAR) products generated by the Alaska Satellite Facility (ASF). 
+
+### Sentinel-1 Bursts
+
+Data from the Sentinel-1 mission that is suitable for use in interferometry has historically been packaged into Single Look Complex (SLC) products. These SLCs include three subswaths, each containing many individual bursts. The framing of the SLCs is not consistent through time, so when using SLCs as the basis for InSAR, datasets do not always fully overlap. 
```

Comment:
"datasets do not fully overlap" doesn't paint the picture in my head, maybe "individual scenes" instead?
```suggestion
Data from the Sentinel-1 mission that is suitable for use in interferometry has historically been packaged into Single Look Complex (SLC) products. These SLCs include three subswaths, each containing many individual bursts. The framing of the SLCs is not consistent through time, so when using SLCs as the basis for InSAR, individual scenes do not always fully overlap. 
```

Reply:
addressed in [this commit](https://github.com/ASFHyP3/hyp3-docs/pull/360/commits/1f1f0c1e82b857c14a8361b21b62b4f855ac2af9)